src/pharmpy/tools/structsearch/drugmetabolite.py

```diff
@@ -4,6 +4,28 @@
 from pharmpy.modeling import add_metabolite, add_peripheral_compartment, set_name
 
 
+def create_base_drug_metaboltie(model: Model) -> Model:
+    """
+    Create a plain metabolite models from an input model. This is used
+    as the base model when searching for candidate drug metabolite model
+
+    Parameters
+    ----------
+    model : Model
+        Input model for search
+
+    Returns
+    -------
+    Model
+        Base model for drug metabolite model search
+
+    """
+    model = add_metabolite(model)
+    model = set_name(model, "plain")
```

Comment:
Perhaps name "base" to match other tools or alternatively "base-metabolite"

Reply:
named "base_metabolite" for clarity of origin